index.js

```diff
@@ -80,6 +80,28 @@ const parseInput = async input => {
 	return input;
 };
 
+const killChecker = async input => {
+	let i = 0;
+
+	return new Promise(resolve => {
+		const checker = setInterval(async () => {
```

Comment:
It would be better to use `setTimeout` and call it recursively. Now, if `processExists` for some reason takes more than 100 ms, it will be called too often.